x-pack/plugins/security/server/authorization/authorization_service.tsx

```diff
@@ -176,8 +179,9 @@ export class AuthorizationService {
     initAPIAuthorization(http, authz, loggers.get('api-authorization'));
     initAppAuthorization(http, authz, loggers.get('app-authorization'), features);
 
-    http.registerOnPreResponse((request, preResponse, toolkit) => {
+    http.registerOnPreResponse(async (request, preResponse, toolkit) => {
       if (preResponse.statusCode === 403 && canRedirectRequest(request)) {
+        const customBrandingValue = await customBranding.getBrandingFor(request, true);
```

Comment:
__optional nit:__ same nit here:
```suggestion
        const customBrandingValue = await customBranding.getBrandingFor(request, /* unauthenticated */ true);
```

__note:__ __But__ shouldn't we use `unauthenticated: false` here? We return `403` when user isn't authorized to access something or perform an action, but the user is still authenticated.